dask_yarn/core.py

```diff
@@ -200,6 +200,12 @@ def __enter__(self):
     def __exit__(self, *args):
         self.close()
 
+    def __str__(self):
+        return "<Dask on YARN cluster {}:{}>".format(
+            self.knit.conf['rm'], self.knit.conf['rm_port'])
+
+    __repr__ = __str__
```

Comment:
Not a big deal, but if you define `__repr__` instead, the default for `__str__` is to fall back to it. No need for defining both:

```python
In [1]: class Foo(object):
   ...:     def __repr__(self):
   ...:         return 'foo'
   ...:

In [2]: f = Foo()

In [3]: f
Out[3]: foo

In [4]: str(f)
Out[4]: 'foo'
```